tests/disas/call-indirect.wat

```diff
@@ -16,7 +16,7 @@
 ;;     gv5 = load.i64 notrap aligned gv3+56
 ;;     sig0 = (i64 vmctx, i64, i32) -> i32 tail
 ;;     sig1 = (i64 vmctx, i32, i64) -> i64 tail
-;;     fn0 = colocated u1:9 sig1
+;;     fn0 = colocated u1610612736:9 sig1
```

Comment:
Maybe print the indexes as hex without `0x` prefix? `format!("u{namespace:x}:{index:x}")`